.github/workflows/build-and-test-differential.yaml

```
@@ -22,6 +22,10 @@ on:
         required: false
         default: ""
         type: string
+      build-pre-command:
+        required: false
+        default: nice -n 19
```

```suggestion
        default: build-pre-command: taskset --cpu-list 0-6
```

Let's keep the default functional too.